### DISCUSSION TITLE
Test prerelease

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#45] [General]
Hi @marchoeppner 
if we plan to open the repo to users for a test phase after frieday, should we make a prerelease?
The current `dev` version looks pretty good. we could push it to `main` and make a `1.0.0.alpha` release.

Thoughts?

**Comment by marchoeppner** [comment]
Fine with me. I think the only to-do would be to update the pipeline version in nextflow.config (sadly, no way to link that to the github release tags automatically, as far as I know)
